cmd/application_scale.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+	"github.com/spf13/cobra"
+)
+
+var processType string
+var processCount int
+
+var appScaleCmd = &cobra.Command{
+	Use:     "scale",
+	Aliases: []string{"change", "modify", "upgrade"},
+	Example: "civo app scale APP-NAME --process-type=web --process-count=3",
+	Short:   "Scale processes of your application",
+	Args:    cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		client, err := config.CivoAPIClient()
+		if err != nil {
+			utility.Error("Creating the connection to Civo's API failed with %s", err)
+			os.Exit(1)
+		}
+
+		findApp, err := client.FindApplication(args[0])
+		if err != nil {
+			utility.Error("App %s", err)
+			os.Exit(1)
+		}
+
```

Comment:
Can we validate the process names? Supported types are:
* web
* worker
* release
* clock

We can error out for any other process type user gives.

Reply:
Okay this is done